src/faebryk/libs/kicad/fileformats_sch.py

```diff
@@ -27,6 +26,12 @@
 
 
 def uuid_field():
+    def gen_uuid():
```

Comment:
Boy... this got to be a big mess. One of the tests had a circular import, which this managed to break.

Reply:
Fixed in #113